assessorportal_cloud_init.tf

```diff
@@ -18,7 +18,9 @@ data "cloudinit_config" "assessorportal_cloud_init_tasks" {
       "${path.module}/cloud-init/efs-mount.tpl.yml", {
         # Just mount the EFS mount target in the first private subnet
         efs_id      = aws_efs_mount_target.target[var.private_subnet_cidr_blocks[0]].file_system_id
+        group       = "vnc"
         mount_point = "/share"
+        owner       = "vnc"
```

Comment:
The only issue I see is that this is a cloud-config file and not a shell script. Thoughts on using an [`aws_ssm_parameter` data source](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ssm_parameter)?

Reply:
I don't love this bit with the `aws_ssm_parameter` data source:
```
The unencrypted value of a SecureString will be stored in the raw [Terraform] state as plain-text.
```

In this case, it wouldn't be that hard to turn this cloud-config file into a shell script.  Or maybe just split out the `runcmd` bits into their own shell script if we don't want to lose the simplicity of using that `mounts` command.

Reply:
PS: I'm not worried about this particular SSM value being in the Terraform state- just more generally about us going down that path and ending up with something more sensitive landing in the state.

Reply:
Yeah I only suggested it given the non-sensitive nature of this value. We could always add a disclaimer that the value is non-sensitive so it's fine if it's in the Terraform state?

Reply:
I suppose that's ok in this case.  Though if it's not sensitive, then maybe it doesn't even belong in SSM?  Though it is convenient to have certain values in SSM if they are accessed by Terraform, cloud-init, and packer scripts.

Reply:
> Though if it's not sensitive, then maybe it doesn't even belong in SSM?

SSM Parameter Store is for storing parameters, secret or otherwise.  There's nothing wrong with storing non-secret parameters there for convenience, although in many cases it may be simpler to store such parameters in, e.g. a `*.tfvars` file.

Reply:
Please enjoy d0f2f37 for a non-static version.

Reply:
Perfecto - nice work.